composer.json

```diff
@@ -14,8 +14,11 @@
   "require-dev": {
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
     "phpcompatibility/php-compatibility": "^9.3",
+    "phpstan/extension-installer": "^1.3.0",
+    "phpstan/phpstan": "^1.10",
```

Comment:
Yes, I thought about that. But I thought this might get us PHPStan updates more quickly by being an explicit dependency. Otherwise we'd need to wait for the package to update, AFAIK.